ui/components/app/custom-spending-cap/custom-spending-cap-tooltip.js

```diff
@@ -35,13 +31,19 @@ export const CustomSpendingCapTooltip = ({
     >
       {tooltipIcon ? (
         <Icon
-          name={ICON_NAMES.DANGER}
+          name={IconName.Danger}
           className="form-field__heading-title__tooltip__warning-icon"
-          size={ICON_SIZES.AUTO}
+          size={IconSize.Auto}
           style={{ 'vertical-align': 'bottom' }}
         />
       ) : (
-        tooltipIcon !== '' && <Icon name={ICON_NAMES.QUESTION} />
+        tooltipIcon !== '' && (
+          <Icon
+            name={IconName.Question}
+            size={IconSize.Auto}
+            style={{ 'vertical-align': 'bottom' }}
```

Comment:
`Auto` does not exist you should not need the style tag 
```suggestion
            size={IconSize.Inherit}
```